src/js/stores/SystemLogStore.js

```diff
@@ -0,0 +1,192 @@
+import PluginSDK from 'PluginSDK';
+
+import {
+  REQUEST_SYSTEM_LOG_ERROR,
+  REQUEST_SYSTEM_LOG_SUCCESS,
+  REQUEST_PREVIOUS_SYSTEM_LOG_ERROR,
+  REQUEST_PREVIOUS_SYSTEM_LOG_SUCCESS,
+  SERVER_ACTION
+} from '../constants/ActionTypes';
+import AppDispatcher from '../events/AppDispatcher';
+import {
+  SYSTEM_LOG_CHANGE,
+  SYSTEM_LOG_REQUEST_ERROR
+} from '../constants/EventTypes';
+import BaseStore from './BaseStore';
+import SystemLogActions from '../events/SystemLogActions';
+import Util from '../utils/Util';
+
+// Max data storage
+const MAX_FILE_SIZE = 250000000;
```

Comment:
Yes! Added a description, and updated the value to something a little bit more describing